### ISSUE TITLE
v1.3 batch_norm layer

### ISSUE THREAD

**Issue by sddi** [tensorflow/tensorflow#16652]
I use the batch norm layer like this:
`def batch_norm_layer(x,train_phase,scope_bn):

	bn_train = batch_norm(x, decay=0.999, center=True, scale=True,
	is_training=True,
	reuse=None, # is this right?
	trainable=True,
	scope=scope_bn)
	bn_inference = batch_norm(x, decay=0.999, center=True, scale=True,
	is_training=False,
	reuse=True, # is this right?
	trainable=True,
	scope=scope_bn)
	z = tf.cond(train_phase, lambda: bn_train, lambda: bn_inference)
	return z`
I don't know in v1.3.0 is the code worked?
I saw the [issue1122](https://github.com/tensorflow/tensorflow/issues/1122#issuecomment-232535426), someone said it would not work well.

thank you in advance.

**Comment by drpngx** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
